src/test/java/com/linecorp/armeria/server/docs/DocServiceTest.java

```diff
@@ -52,17 +53,24 @@
     @Override
     protected void configureServer(ServerBuilder sb) {
         final ThriftService helloService = new ThriftService(HELLO_SERVICE_HANDLER);
-        final ThriftService fooService = new ThriftService(mock(FooService.AsyncIface.class));
-        final ThriftService cassandraService = new ThriftService(mock(Cassandra.AsyncIface.class));
+        final ThriftService fooService = new ThriftService(mock(FooService.AsyncIface.class),
+                                                           SerializationFormat.THRIFT_BINARY,
+                                                           EnumSet.of(SerializationFormat.THRIFT_BINARY));
+        final ThriftService cassandraService = new ThriftService(mock(Cassandra.AsyncIface.class),
+                                                                 SerializationFormat.THRIFT_BINARY,
+                                                                 EnumSet.of(SerializationFormat.THRIFT_BINARY));
+        final ThriftService cassandraServiceDebug =
+                new ThriftService(mock(Cassandra.AsyncIface.class),
+                                  SerializationFormat.THRIFT_TEXT,
+                                  EnumSet.of(SerializationFormat.THRIFT_TEXT));
```

Comment:
The mime-type parameter behavior is tested by helloService and hbaseService, I left cassandra split just to also test the case where the paths for debug and normal are split. I think it'd actually be harder to not support this behavior, so it seems nice that we can handle both use cases.